packages/salesforcedx-vscode-apex/snippets/apex.json

```
@@ -410,5 +409,54 @@
             "}"
         ],
         "description": "Create a mock for an http callout for apex testing"
+    },
+    "Is Blank": {
+        "prefix": "isb",
+        "body": [
+            "String.isBlank(${0})"
+        ],
+        "description": "String isBlank"
+    },
+    "Is Not Blank": {
+        "prefix": "isnb",
+        "body": [
+            "String.isNotBlank(${0})"
+        ],
+        "description": "String Not isBlank"
+    },
+    "local variable": {
+        "prefix": "localVariable",
+        "body": [
+            "${1:Object} ${2:name} = ${3:};"
```

This would be clearer if there is a value parameter to replace.

```suggestion
            "${1:Object} ${2:name} = ${3:value};"
```